internal/oci/container.go

```diff
@@ -79,6 +79,7 @@ type Container struct {
 	restore            bool
```

Comment:
@hasan4791 Please add a footer in the commit message to reference the issue this PR is fixing. See [contribution guidelines](https://github.com/cri-o/cri-o/blob/main/CONTRIBUTING.md#submitting-pull-requests). 

For example:
```
Closes #5653
```

In addition, consider adding to the commit message what this commit is fixing and why.

Reply:
Sure, will keep it in mind